tools/depends/target/ffmpeg/FFMPEG-VERSION

```diff
@@ -1,4 +1,4 @@
 LIBNAME=ffmpeg
-BASE_URL=https://github.com/xbmc/FFmpeg
-VERSION=4.0.4-Leia-18.4
+BASE_URL=https://github.com/jaylinski/FFmpeg
```

Comment:
This has to be changed back to `https://github.com/xbmc/FFmpeg` as soon as an appropriate tag was created.